### DISCUSSION TITLE
What's the correct way to use combinatorials for negative prompts?

### DISCUSSION THREAD

**Discussion by TiagoTiago** [adieyal/sd-dynamic-prompts#66] [Q&A] [answered]
****

**Comment by adieyal** [comment, marked as answer]
Unfortunately you can't. Automatic1111 doesn't allow multiple negative prompts. To get around this we change the negative prompt for every batch which helps with random but not with combinatorial.

**Reply by TiagoTiago** [reply]
Oh, it's not just adding and removing text to the single text field we see on the page for the positive and negative prompts respectively?

**Reply by adieyal** [reply]
auto1111 has now changed this behaviour. There is an issue to implement this which you can follow here: https://github.com/adieyal/sd-dynamic-prompts/issues/70
